OMCompiler/README-Windows-WSL.md

```diff
@@ -44,8 +50,8 @@ git clone --recursive https://openmodelica.org/git-readonly/OpenModelica.git Ope
 ```bash
 cd OpenModelica
 autoconf
-./configure CC=clang CXX=clang++ -without-omc -with-cppruntime
-make -j4    # Or 4 with the number of cores you have
+./configure CC=clang CXX=clang++ --without-omc --with-cppruntime
+make -j4    # Or replace 4 with the number of threads you have
```

Comment:
I strongly recommend the number of physical cores since hyperthreading mostly uses up all your RAM

Reply:
Thanks, I've update it.